content/22.vaccines.md

```diff
@@ -192,16 +192,66 @@ As of December 11, 2020, the United States FDA approved this vaccine under an em
+Genetic material from the target virus of interest is delivered using a second virus as a vector.
+The vaccine then uses the host machinery to construct antigen(s) from the transported genetic material, for which the body synthesizes antibodies in response.
+The genetic content of the vector virus is often altered to prevent it from replicating, but replication-competent viruses can also be used under certain circumstances [@doi:10.1016/j.coi.2016.05.014].
+Viral-vector vaccines are able to induce both an antibody and cellular response; however, the response is limited due to the immunogenicity of the viral vector used [@doi:10.1016/j.coi.2011.03.006; @doi:10.1038/mt.2009.130].
+Various viral-vector platforms including poxviruses [@doi:10/cnw6vw; @doi:10.4161/hv.28974], adenoviruses [@doi:10.1038/mt.2009.130], and vesicular stomatitis viruses [@pmid:10196265; @doi:10.1016/j.molmed.2004.03.003] are being developed,
+An important consideration in identifying potential vectors is the immune response to the vector.
+Both the innate and adaptive immune responses can potentially respond to the vector, limiting the ability of the vaccine to transfer information to the immune system [@doi:10.1038/gt.2009.148].
+Different vectors are associated with different levels of reactogenicity; for example, adenoviruses elicit a much stronger innate immune response than replication deficient adeno-associated viruses derived from parvoviruses [@doi:10.1038/gt.2009.148].
+Additionally, using a virus circulating widely in human populations as a vector presents additional challenges because vaccine recipients may already have developed an immune response to the vector [@doi:10.3390/vaccines2030624].
+
+There are several viral vector vaccines that are available for veterinary use [@doi:10.3390/vaccines8040680], but prior to the COVID-19 pandemic, only one viral vector vaccine was approved by the FDA for use in humans.
+This vaccine is vectored with a recombinant vesicular stomatitis virus and targeted against the ebola virus [doi:10.1016/j.cell.2020.03.011].
+Additionally, several phase I and phase II clinical trials for other vaccines are ongoing [@doi:10.1128/CVI.00298-16].
+This technology is currently being explored for its potential against numerous infectious diseases including malaria [@doi:10.1016/j.ymthe.2016.11.003; @doi:10.1016/j.vaccine.2006.07.035], ebola [@doi:10.1056/NEJMoa1410863; @doi:10.1093/infdis/jir349; @doi:10.1586/14760584.2014.885841], human immunodeficiency virus (HIV) [@doi:10.1097/COH.0b013e328363d389; @doi:10.1016/j.virol.2014.09.004].
+The prior MERS and SARS initiated interest in the application of viral vector vaccines to human coronaviruses [@doi:10.3390/v12080861], but efforts to apply this technology to these pathogens had not yet led to a successful vaccine candidate.
```

Comment:
What initiated interest? prior epidemics? Or just that there was interest in using viral vectors for these viruses?